appengine-managed-runtime/src/test/java/com/google/apphosting/vmruntime/VmApiProxyDelegateTest.java

```diff
@@ -194,6 +196,11 @@ private void callDelegateWithOneError(
     byte[] result = null;
     final Double timeoutInSeconds = 10.0;
 
+    Logger logger = Logger.getLogger(VmApiProxyDelegate.class.getName());
+    Level level = logger.getLevel();
+    if (!logger.isLoggable(Level.FINE)) {
+      logger.setLevel(Level.SEVERE);
```

Comment:
If somebody has set logging to INFO, that is kind of the default, so normally they would see INFO, WARNING and SEVERE, which would mean they would see stack traces.   So for this unit test, I'm saying that if the level is INFO (ie it would not log a FINE), then I'm assuming that is default logging and by default I think the stack traces should be suppressed.

If the logging has been set so FINE or above is visible, somebody has asked for details and thus I don't change the logging.